x11-misc/e16-keyedit/e16-keyedit-0.10.ebuild

```diff
@@ -6,20 +6,14 @@ EAPI=8
 DESCRIPTION="Key binding editor for Enlightenment DR16"
 HOMEPAGE="https://www.enlightenment.org https://sourceforge.net/projects/enlightenment/"
 SRC_URI="https://downloads.sourceforge.net/enlightenment/${P}.tar.xz"
-KEYWORDS="~amd64 ~x86"
 
 LICENSE="MIT-with-advertising"
 SLOT="0"
+KEYWORDS="~amd64 ~x86"
 
-BDEPEND="
-	virtual/pkgconfig
-"
-DEPEND="
-	x11-libs/gtk+:3
-"
-RDEPEND="${DEPEND}
-	!x11-misc/e16keyedit
-"
+BDEPEND="virtual/pkgconfig"
+DEPEND="x11-libs/gtk+:3"
+RDEPEND=${DEPEND}
```

Comment:
Missing quotes.

Reply:
If he wants to use that style in his own ebuilds, that's fine, but in general, please include it.